src/benchmark_register.cc

```diff
@@ -114,7 +114,14 @@ bool BenchmarkFamilies::FindBenchmarks(
   // Make regular expression out of command-line flag
   std::string error_msg;
   Regex re;
-  if (!re.Init(spec, &error_msg)) {
+  std::string prefix("-");
+  std::string temp_spec = std::string(spec);
```

Comment:
given you have to take a copy, it might be better to just pass `spec` in by value instead of const reference.

Reply:
Done :)